model-optimizer/mo/graph/port.py

```diff
@@ -294,13 +294,13 @@ def get_tensor_names_list(attrs):
         assert self.type != 'in', "Can't get tensor names for input port at {} node".format(self.node.name)
 
         fw_names = []
-        if self.node.graph.stage == 'front':
+        if self.node.graph.stage is None or self.node.graph.stage == 'front':
```

Comment:
Do we really need a check that `self.node.graph.stage is None`? In the MO code we always have valid value for the `stage` attribute: either "front" or "middle".

Reply:
Yes, this check is no longer needed in current solution. There were problems with previous implementations.